shotover-proxy/src/frame/cassandra.rs

```diff
@@ -320,7 +284,10 @@ impl CassandraFrame {
 
 #[derive(PartialEq, Debug, Clone)]
 pub enum CassandraOperation {
-    Query { query: CQL, params: QueryParams },
+    Query {
+        query: Box<CassandraStatement>,
+        params: QueryParams,
+    },
     Result(CassandraResult),
     Error(ErrorBody),
     // operations for protocol negotiation, should be ignored by transforms
```

Comment:
Should be ignored by *most* transforms

Reply:
Resolved in https://github.com/shotover/shotover-proxy/pull/679